src/librustc_mir/transform/mod.rs

```diff
@@ -229,6 +231,8 @@ fn mir_const(tcx: TyCtxt<'_>, def_id: DefId) -> Steal<Body<'_>> {
         None,
         MirPhase::Const,
         &[&[
+            // MIR-level lints.
+            &check_packed_ref::CheckPackedRef,
             // What we need to do constant evaluation.
```

Comment:
I don't even know what this means... I think this is a leftover from the pre-miri MIR based const evaluator. I think we could just merge the entire `mir_const` query into `mir_validated`

Reply:
Hm, maybe we can, I'll leave that to someone else.^^